keywords.robot

```diff
@@ -2978,3 +2978,26 @@ Should Contain All
     FOR    ${substring}    IN    @{substrings}
         Should Contain    ${string}    ${substring}
     END
+
+Flash SD Wire
+    [Documentation]    This keyword flashes the sd wire that's connected to
+    ...    the provided RTE IP.
+    Open Connection And Log In
+
+    Variable Should Exist    ${file_bmap}
+    Variable Should Exist    ${file_gz}
+
+    SSHLibrary.Put File    ${file_bmap}    /data/    scp=ON
+    SSHLibrary.Put File    ${file_gz}    /data/    scp=ON
+
+    # sonoff turn off power should be here
+
+    SSHLibrary.Execute Command    sd-mux-ctrl -e=sd-wire_01-80 --ts
+    SSHLibrary.Execute Command    umount /dev/sda*
+    SSHLibrary.Execute Command    bmaptool copy --bmap /data/${file_bmap} /data/${file_gz} /dev/sda
+
+    SSHLibrary.Execute Command    sd-mux-ctrl -e=sd-wire_01-80 --dut
+
+    # sonoff turn on power should be here
+
+    SSHLibrary.Close All Connections
```

Comment:
@WiktorG351 this should be moved to test suite teardown, because can break test-cases/test-suites